src/Files.App/Interacts/IBaseLayoutCommandImplementationModel.cs

```diff
@@ -98,6 +98,10 @@ public interface IBaseLayoutCommandImplementationModel : IDisposable
 
 		Task CompressIntoArchive();
 
+		Task CompressIntoZip();
+
+		Task CompressIntoSevenZip();
+
```

Comment:
Can we merge them into `CompressIntoArchive` which uses a parameter to decide the archive format?

Reply:
I prefer to use multiple methods because the role of this parameter would not be intuitive. CompressIntoArchive displays a dialog with several options. CompressIntoZip directly creates the archive without any options. Optionally, we can group CompressIntoZip and CompressIntoSevenZip together. But these functions should correspond to methods (see below).